readme.md

```diff
@@ -1 +1,64 @@
-# Hello Microverse project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Microverse Linters Project
+
+> This is a project that equips you with the necessary knowledge on linters configurations, making and editing a pull request and adding a professional readme to your project.
+
+## Built With
+
+- HTML and CSS
+- Linters
+- VS studio code, GitHub.
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+## Getting Started
+
+**In order to use this project, you have to create a .githhub/workflow folder on your root directory and add this file 'linters.yml'. Then checkout to your feature branch and compare with your main branch and then check linters for error on your feature branch.**
+
+---
+
+To get a local copy up and running follow these simple example steps.
```

Comment:
- [ ] Please 🙏🏽 replace the information with your instructions to use your repository, for example:
- Clone the repo and open it in your browser.